tests/cases/fourslash/formattingObjectLiteralOpenCurlyNewlineAssignment.ts

```diff
@@ -0,0 +1,47 @@
+/// <reference path='fourslash.ts' />
+
+//// 
+//// var clear = {};
+//// clear =

```

Comment:
I liked the `||` example you mentioned offline - add a test for that too?

Reply:
I will take the opportunity to make this test more minimal - it's copied out of another test and I don't like how redundant it is